src/configfile.ts

```diff
@@ -215,6 +218,26 @@ interface ChromeConfig extends BrowserConfigBase {
    * @TJS-minimum 1
    */
   cpuThrottlingRate?: number;
+
+  /**
+   * Optional config to turn on performance tracing.
+   */
+  trace?: TraceConfig|boolean;
+}
+
+/**
+ * Configuration to turn on performance tracing
+ */
+interface TraceConfig {
+  /**
+   * The tracing categories the browser should log
+   */
+  categories?: string[];
```

Comment:
Again here maybe a link to https://chromium.googlesource.com/chromium/src/+/master/base/trace_event/builtin_categories.h (or a better link if you know one).

Reply:
I included the link to the source in this comment, but just realized I did not add it to the readme. Should I add a link to the source file there too?